.taskcluster.yml

```diff
@@ -20,8 +20,58 @@ tasks:
     extra:
       github:
         events:
+          - pull_request.opened
           - pull_request.synchronize
```

Comment:
Ok, we need both opened and synchronize.
opened for trigerring test on pull request opening,
synchronize when you push new commits.